team_structure.md

```diff
@@ -0,0 +1,95 @@
+# Team Structure
+
+## Org Chart & Interfaces
+
+```mermaid
+flowchart TD
+    CEO{CEO};
+    CTO{CTO};
+    PO{Product Owner};
+    team{voiio};
+    TM{tech. Manager};
+    TL{dep. Head};
+    CEO -- compnay OKR --> CTO;
+    CTO -- product OKR --> PO;
+    CTO <-. platform architecture .-> TM;
+    PO -- project OKR --> TM;
+    TM <-- project collab. --> TL;
+    PO <-- roadmap planing --> TL;
+    PO -- product update --> team;
+```
+
+## Artifacts & Ownership
+
+```mermaid
+flowchart LR
+    CEO{CEO};
+    CTO{CTO};
+    PO{PO};
+    TM{TM};
+    companyVision[[Company Vision]];
+    productVision[[Product Vision]];
+    productRoadmap[[Product Roadmap]];
+    companyObjectives[[Company Objectives]];
+    companyKeyResults[[Company Key Results]];
+    productObjectives[[Product Objectives]];
+    productKeyResults[[Product Key Results]];
+    projectKeyResults[[Project Key Results]];
+    projectObjectives[[Project Objectives]];
+    project[[Project]];
+    CEO -. owns .- companyVision;
+    CEO -. sets .- companyObjectives;
+    CTO -. delivers .- companyKeyResults;
+    CTO -. owns .- productVision;
+    CTO -. set .- productObjectives;
+    PO -. delivers .- productKeyResults;
+    PO -. owns .- productRoadmap;
+    PO -. sets .- projectObjectives;
+    TM -. delivers .- projectKeyResults;
+    TM -. owns .- project;
+```
+
+### Visions
+
+A clear vision needs to be set for the company and all departments. Those documents
+need to be shared with all employees.
+
+### OKR
+
+_ORK = Objective Key Result_
+
+We use OKRs as a bidirectional communication, with goal alignment in the form of
+objectives and impactful delivery in form of key results. They must be agreed upon
+by both the team and manager.
+
+ORKs are communicated within the company to identify synergies or conflicts of interest.
+
+#### Objectives
```

Comment:
On a long term it would make sense how objectives should look like. Here is why:

What are outcome-focused objectives and why do they matter? While an output is what a team does, outcomes reflect the change that occurs based on what a team does. In many cases, building outcome-focused objectives requires you to shift from an activities-based mindset to an outcome-based one.